ChangeLog.md

```diff
@@ -1,3 +1,40 @@
+# 1.4.1 -> 1.4.2
+
+### General
+* Restored support for Windows 7
+* Restored support for 32-bit builds
+* Using alt-tab to switch to another application will no longer activate in-game console/chat
+* added nwcfollowthewhiterabbit cheat: the currently selected hero permanently gains maximum luck.
+* added nwcmorpheus cheat: the currently selected hero permanently gains maximum morale.
+* added nwcoracle cheat: the puzzle map is permanently revealed.
+* added nwctheone cheat: reveals the whole map, gives 5 archangels in each empty slot, unlimited movement points and permanent flight to currently selected hero
```

Comment:
I would probably add "changed frequency of client checking if server finished startup from 1s to 0.2s"